languages/go/exercises/concept/strings/string.go

```diff
@@ -0,0 +1,16 @@
+package strings_concept
```

Comment:
I would have liked to call this file `strings.go` like the folder. There is something weird going on however with a file named `strings.go`. Can't run the tests from my IDE then. Don't want that to happen to a student.

Reply:
Package strings would maybe conflict with the std library's homonym package, but I'm surprised that the name of the file gives problems too. Just for curiosity what was your error? In which IDE?

Reply:
IDE is `Goland`, and when running the tests it complained that functions didn't exist at all.  As if the file with the stubs was gone completely. renaming the file to `string.go` the functions could be found all of a sudden. I didn't dig deeper into it, after confirming the `go test` from the terminal had no issues with it.

Reply:
Hmmm... can't reproduce the error. Maybe I did something weird.

I will rename them to `strings.go` and `strings_test.go`.